GrocifyApp.API/Models/Mapper/AutoMapperProfile.cs

```diff
@@ -23,6 +23,10 @@ public AutoMapperProfile()
             CreateMap<ShoppingListRequestModel, ShoppingList>().ReverseMap();
```

Comment:
can you please go to Repository.cs, and throw NotFoundException here:

![image](https://github.com/Catarina98/Grocify/assets/36730010/f20a9a0a-a7b8-4e4c-98ce-d2dc585ef7fd)

Because when I try to delete an entity that does not exist, the error is "Something went wrong" and should be "Entity not found"